ReadMe.md

```diff
@@ -1,18 +1,197 @@
-# Hello-Microverse_Project
 
-Hello-Microverse_project is a beginner level project at Microverse, that requires requires the student to display hello on a web page with simple styling. 
 
-# Built With
-HTML
-CSS
+# 📗 Table of Contents
 
-# Key Features
-1. Linter for HTML and CSS: checks errors and organise code
-2. gitignore: to keep some directories from being committed
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+
+# 📖 Hello-Microverse_project <a name="about-project"></a>
+
+
+**Hello-Microverse_project** is a beginner level project at Microverse. The project requires a student to add "Hello Microverse" to a webpage and apply some basic styling to it. It also requires the use of linters, .gitignore file and other best practices.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+  <summary>Linters</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Linters**
+- **.gitignore file**
+- ****
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: A code editor(e.g VS code)
+
+### Setup
+
+Clone this repository to your desired folder:(e.g git clone git@github.com:myaccount/my-project.git)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Install
+
+Install this project with:
+
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
```

Comment:
Please remove this and replace it with how a user can `run` your project on his local machine. Example
`To run the project, open the index.html file with a browser or use a live server with your favorite code editor`